quantus/helpers/enums.py

```diff
@@ -0,0 +1,61 @@
+from enum import Enum
```

Comment:
I guess, this is more a question of test. What do you think of using `typing.Literal`? E.g., 

```
from typing import Literal 

DataType = Literal["text", "image", "time-series"]
```

No real benefit, though 🤷‍♂️

Reply:
hmm! Literal was introduced in Python3.8, so for our package, enum might be better?

Reply:
then the question is closed 🤷